### PR TITLE
exam commit

### PR DESCRIPTION
**Pull request by hobbitJS**

### CODE REVIEW

File: src/App.vue [hunk 1]
```diff
@@ -0,0 +1,29 @@
+<template>
+  <div id="app">
+    <div id="nav">
+      <router-link to="/">Home</router-link> |
+      <router-link to="/issues">Issues</router-link>
```
Comment (rapi) [inline src/App.vue:5]: use router name not href

File: src/store.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
+import Vue from 'vue'
+import Vuex from 'vuex'
+import axios from 'axios'
+
+Vue.use(Vuex)
+
+let axiosClient = axios.create({
```
Comment (rapi) [inline src/store.js:7]: axios thins should be in different folder

File: src/components/IssueDetail.vue [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    <div class="card-header">
+      <h2>{{issue.title}} #{{issue.number}}</h2>
+      <p>
+        <h5>
+          <span class="badge badge-secondary">{{issue.state}}</span>
+        </h5>
+      </p>
+    </div>
+    <div class="card-body">
+      <p class="card-text">{{issue.body}}</p>
```
Comment (rapi) [inline src/components/IssueDetail.vue:12]: Lint it

File: src/store.js [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+    'SET_LIST': (state, list) => {
+      state.list = list
+    },
+    'SET_SELECTED_ISSUE': (state, issue) => {
+      state.selectedIssue = issue
+    }
+  },
+  actions: {
+    'SET_LIST': ({ commit }, list) => {
+      axiosClient.get('repos/vuejs/vue/issues')
```
Comment (rapi) [inline src/store.js:26]: better to separate it

File: src/components/IssueDetail.vue [hunk 5]
```diff
@@ -0,0 +1,36 @@
+<template>
+  <div class="card">
+    <div class="card-header">
```
Comment (rapi) [inline src/components/IssueDetail.vue:3]: Make loading or if issue exit

**Review by rapi — COMMENTED** [review]
Lint problem and some structure problem

File: src/components/IssueDetail.vue [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+import { mapActions, mapGetters } from 'vuex'
+
+export default {
+  name: 'issueDetail',
+  computed: {
+    ...mapGetters({
+      issue: 'GET_ISSUE_BY_ID'
+    })
+  },
+  mounted () {
```
Comment (gregurco) [inline src/components/IssueDetail.vue:27]: better to use `created`

**Pull request merged**